src/utils/components/ImageUpload.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <div class="full-width">
+    <Croppa
```

Comment:
oh no, another croppa :( well, can replace them in a batch at some point...

Reply:
I did actually spend a bit of time on this evaluating other options, but was going to be more work. I put it in a separate component to at least isolate it a little bit from the ActivityEdit form...

I liked https://advanced-cropper.github.io/vue-advanced-cropper/ ... but it doesn't do the uploading/selecting/etc, so much more work to make it a replacement for vue-croppa, but maybe worth it at some point.

I've periodically been wondering about having a generic image endpoint, and then storing referenced ids in the db for images (so it can upload whilst continuing to edit the form), we could potentially use a temporary images table though, and copy it over after submit (so still use the existing versatile image fields... ), might be a nice solution... but yeah, not now :)